convert2rhel/toolopts.py

```diff
@@ -304,9 +361,24 @@ def _process_cli_options(self):
 
         warn_on_unsupported_options()
 
-        parsed_opts = self._parser.parse_args()
+        args = []
+        argv = sys.argv[1:]
+
+        # algorithm function to properly organize all CLI args
+        argv = _add_default_command(argv)
+
+        has_subcommand = False
+        for index, argument in enumerate(argv):
+            if argument in ("convert", "analyze") and argv[index - 1] not in ARGS_WITH_VALUES:
+                # subcommand is present in argv list
+                has_subcommand = True
+                break
+
+        if not has_subcommand:
+            args.insert(0, "convert")
```

Comment:
I don't think this code is needed anymore, especially that it looks very identical as the code inside the `_add_default_command` function. Probably we can get this removed and rely on the function above to do this job.

Reply:
What about this diff as a suggestion? 

```diff
diff --git a/convert2rhel/toolopts.py b/convert2rhel/toolopts.py
index 5b60469..ea391c5 100644
--- a/convert2rhel/toolopts.py
+++ b/convert2rhel/toolopts.py
@@ -361,25 +361,12 @@ class CLI(object):
 
         warn_on_unsupported_options()
 
-        args = []
-        argv = sys.argv[1:]
-
         # algorithm function to properly organize all CLI args
-        argv = _add_default_command(argv)
-
-        has_subcommand = False
-        for index, argument in enumerate(argv):
-            if argument in ("convert", "analyze") and argv[index - 1] not in ARGS_WITH_VALUES:
-                # subcommand is present in argv list
-                has_subcommand = True
-                break
-
-        if not has_subcommand:
-            args.insert(0, "convert")
-
-        args.extend(argv)
-        parsed_opts = self._parser.parse_args(args)
+        argv = _add_default_command(sys.argv[1:])
+        parsed_opts = self._parser.parse_args(argv)
 
+        print(parsed_opts)
+        sys.exit(0)
         if parsed_opts.debug:
             tool_opts.debug = True
```

Reply:
Yes I will remove this